lib/fxpay/index.js

```diff
@@ -93,23 +92,37 @@
         // as the click handler. This avoids popup blockers.
         opt.paymentWindow = utils.openWindow();
       } else {
-        log.info('web flow will use client provided payment window');
+        settings.log.info('web flow will use client provided payment window');
         utils.reCenterWindow(opt.paymentWindow,
                              settings.winWidth, settings.winHeight);
       }
     }
 
+    function closePayWindow() {
+      if (opt.paymentWindow && !opt.paymentWindow.closed) {
+        if (opt.managePaymentWindow) {
+          opt.paymentWindow.close();
+        } else {
+          settings.log.info('payment window should be closed but client ' +
+                            'is managing it');
+        }
+      }
+    }
+
     settings.adapter.startTransaction({productId: productId},
                                       function(err, transData) {
       if (err) {
```

Comment:
Yeah - I thought that might be the case just thought I'd mention it so we could know it's covered one way or the other.